tasks/main.yml

```diff
@@ -31,20 +31,20 @@
     path: "{{ walle_bashrc }}"
     state: touch
 
-- name: Add env variables to WallE .bashrc
+- name: Add walle_env_vars to walle_bashrc
   ansible.builtin.lineinfile:
     path: "{{ walle_bashrc }}"
     regexp: "^export {{ item.key }}="
     line: 'export {{ item.key }}="{{ item.value }}"'
   with_items: "{{ walle_env_vars }}"
 
-- name: Add env variables for user deletion (WallE)
+- name: Extend walle_bashrc with user-defined walle_extra_env_vars
```

Comment:
The env vars in the list are merged right? Because if `walle_extra_env_vars` does not overwrite the `walle_env_vars`, I would think, these two tasks might overwrite each other on every run, which works, but is not ideal, because it would show `changed` in the ansible output

Reply:
Yes good point, not ideal! I just changed walle_env_vars to a dict so they can be overridden easily before templating. I think that's a cleaner way to do it?